CakePHP/tests/ruleset.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<ruleset name="CakePHP">
+ <description>CakePHP coding standard</description>
+
+ <rule ref="PSR2"/>
```

Comment:
In CakePHP/ruleset.xml many sniffs are ignored for `*/tests/*`. Since the test files for codesniffer are also under `tests` folder many sniffs get ignored when running testsuite for codesniffer itself which is not what we want.